lib/nexmo_markdown_renderer/filters/code_filter.rb

```diff
@@ -6,12 +6,12 @@ def call(input)
           config = YAML.safe_load($1)
     
           if config['config']
-            configs = YAML.load_file("#{ENV['DOCS_BASE_PATH']}/config/code_examples.yml")
+            configs = YAML.load_file("#{Nexmo::Markdown::Config.Nexmo::Markdown::Config.docs_base_path}/config/code_examples.yml")
```

Comment:
@benhayehudi i think you have one extra `Nexmo::Markdown::Config`

Reply:
fixed!